week3/word.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by julia on 6/8/17.
+//
+
+#include "word.h"
```

Comment:
I think the idea is to have a class Dictionary, created with a list of words or adding one at a time, that has the two methods is_word & is_prefix.
Since this is not the focus of the problem, having two sets inside this class for words & prefixes is fine.